DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<p>It might be helpful to illustrate u-band dropouts using a figure showing the spectrum of an LBG with filter profiles overlaid (it's OK if they're filters from another system, as long as it illustrates the point).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Ok I added a link to a blogpost about it (where there's a figure and a video plus extra blurbs of explanation)</p>